src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -638,5 +653,16 @@ private static void Cleanup(HashSet<string> artifactsToCleanup)
                 }
             }
         }
+
+        private static void LogProgress(ILogger logger, in StartedClock globalChronometer, int totalBenchmarkCount, int benchmarksToRunCount)
+        {
+            int executedBenchmarkCount = totalBenchmarkCount - benchmarksToRunCount;
+            double avgSecondsPerBenchmark = globalChronometer.GetElapsed().GetTimeSpan().TotalSeconds / executedBenchmarkCount;
+            TimeSpan fromNow = TimeSpan.FromSeconds(avgSecondsPerBenchmark * benchmarksToRunCount);
+            DateTime estimatedEnd = DateTime.Now.Add(fromNow);
+            string message = $"// ** Remained {benchmarksToRunCount} ({(double)benchmarksToRunCount / totalBenchmarkCount:P1}) benchmark(s) to run." +
+                $" Estimated finish {estimatedEnd:yyyy-MM-dd H:mm} ({fromNow:%h}h {fromNow:%m}m from now) **";
```

Comment:
if `fromNow` is 25 hours, `{fromNow:%h}h {fromNow:%m}m` prints `1h 0m`.

Reply:
@AndreyAkinshin you are right. I gave up on the smart string formats and used `TotalHours` instead. PTAL